pkgs/tools/networking/networkmanager_dmenu/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, glib, fetchFromGitHub, networkmanager, python3Packages
+, gobjectIntrospection, dmenu }:
+
+let inherit (python3Packages) python pygobject3;
+in stdenv.mkDerivation rec {
+  name = "networkmanager_dmenu-${version}";
+  version = "unstable-2017-04-13";
```

Comment:
I am open to your suggestion. I thought having the version defining the unstable state is what the previous suggestion was.